base/compiler/ssair/ir.jl

```diff
@@ -318,7 +318,7 @@ function is_relevant_expr(e::Expr)
                       :gc_preserve_begin, :gc_preserve_end,
                       :foreigncall, :isdefined, :copyast,
                       :undefcheck, :throw_undef_if_not,
-                      :cfunction, :method,
+                      :cfunction, :method, :pop_exc,
```

Comment:
I'm happy to change the symbol used in the AST.

For the C code I'm not so sure — names like `jl_exception_stack_t` and `jl_exception_stack_bt_size` get a bit tiresome after a while. When spelling it out is too verbose it can be useful to make a new noun, so I could compress all `exc_stack` into `excstack` in the C code and just be consistent in the use of that. I tried to avoid using `ex` as an abbreviation because that commonly stands for "expression".

Reply:
@vtjnash would you also prefer all the C code to also use `exception_stack` (at least for types and function/macro names)?

Obviously after massaging this PR for a few months I've gotten rather used to my "new noun" `exc_stack` and I think it's a good tradeoff (though possibly better a single word `excstack`). But renaming is easy and I can use `exception_stack` everywhere if there's a consensus.